UnityProject/Assets/ScriptableObjects/Cargo/Bounties/MiningBounties/Metal.asset

```diff
@@ -0,0 +1,20 @@
+%YAML 1.1
```

Comment:
pretty sure we have already a ``Metal `` trait that can be used to identify the material

Reply:
nevermind just read the path of the file